frameworks/kotlin/src/main/kotlin/helloWorld.kt

```diff
@@ -0,0 +1,14 @@
+package codewars
+
+// https://github.com/JetBrains/kotlin-examples/blob/master/gradle/hello-world/src/main/kotlin/demo/helloWorld.kt
+fun getGreeting(): String {
+    val words = mutableListOf<String>()
+    words.add("Hello,")
+    words.add("world!")
+
+    return words.joinToString(separator = " ")
+}
```

Comment:
A more kotlinic way:

```kotlin
val greeting: String get () = listOf("Hello,", "world!").joinToString(separator = " ")
```

The above code generates the equivalent jvm bytecode